RealmSwift

```diff
@@ -0,0 +1 @@
+RealmSwift-swift1.2
```

Comment:
Is it necessary to check the symlinks in? The `Tests` one in particular looks like it'll be recreated before it's ever used, but perhaps this one already needs to exist before the Xcode project can be used?